streamlit-app/app.py

```diff
@@ -0,0 +1,70 @@
+import streamlit as st
+import os
+
+from ultralytics import YOLO
+
+from src.video_object import VideoObject
+from src.team_detector import TeamDetector
+
+
+def main():
+    st.set_page_config(page_title="Frisbee dashboard", layout="wide")
+    st.title("Tactical  dashboard")
+
+    with st.sidebar:
+        st.title("Configuration")
+
+        predict_type = st.radio("What type of prediction would you like to see?",
+                                ["default", "team"],
+                                 captions=["Default YOLO [player,referee, disc]", "Team Detection [dark, bright]"])
+
+        st.subheader("Video uploader")
+        uploaded_video = st.file_uploader("Choose a video file", type=["mp4"])
+        st.subheader("Model uploader")
```

Comment:
A thought regarding the model: In the final version of our web app, we should ship the app with the 'best' model (either in source control, or e.g. in a GCS bucket). That's because the app's intended audience is non-technical (and most used stories would start with "As an Ultimate team coach, I would like...")
Overriding the model (either by uploading the model file, or replacing it in a bucket) would be an advanced option.